src/main/java/org/dwcj/controls/DateEditBox.java

```diff
@@ -47,6 +63,11 @@ public boolean getBeep(){
         return false;
     }
 
+    /**
+     * This method returns the caret position in the DateEditBox control.
+     * 
+     * @returns Returns the position of the caret in the BBjInputD control.
+     */
     public int getCaretPosition(){
         try {
```

Comment:
for all these methods it'd be probably better style to check for null in the beginning. The try will do but it's not good style.

Reply:
Agreed, I missed the null check in this function, and will go back and add it!